src/menu.ts

```diff
@@ -336,7 +336,7 @@ export class MenuRange<C extends Context> {
      * @param text The text to display
      * @param url An HTTPS URL of a Web App to be opened with additional data
      */
-
+    webApp(text: MaybeDynamicString<C>, url: string): this;
```

Comment:
There's no need for a function overload. Feel free to replace string by maybe dynamic string, as the latter one includes the former.

Reply:
Done. Could you make the url dynamic?